README.md

```diff
@@ -54,4 +54,9 @@ Finally, if you are sure no one else had the issue, follow the [ISSUE_TEMPLATE](
 
 ### Tip
 
-Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process
+1 > Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process
+
+2 > On some ESP32 development boards, When you trying to upload a new sketch, the Arduino IDE gives you a fatal error. ( like A fatal error occurred: Failed to connect to ESP32: Timed out waiting for packet header) This means your ESP32 board does not goes to automatically into flashing/uploading mode. 
+
+Solution:  To make flashing automatically, You have to connect a 10 uF electrolyte capacitor between the EN pin and GND pin.
```

Comment:
Solution: To make flashing automatically, you have to connect a 10 uF capacitor between the EN and GND pins. This capacitor value could be lower (> 2.2 uF) if you don't have this part available.

EN <----||---- GND

The capacitor can be electrolytic, ceramic or tantalum. Using lower values like 2.2uF and 4.7 uF is also possible in most of the cases.